tests/unit/models/citation-test.ts

```diff
@@ -0,0 +1,12 @@
+import { run } from '@ember/runloop';
+import { setupTest } from 'ember-osf-web/tests/helpers/osf-qunit';
+import { module, test } from 'qunit';
+
+module('Unit | Model | citation', hooks => {
+    setupTest(hooks);
+
+    test('it exists', function(assert) {
+        const model = run(() => this.owner.lookup('service:store').createRecord('citation'));
+        assert.ok(!!model);
```

Comment:
Not going to touch this. 😄

Reply:
Generally, I'm down for preferring `Boolean(foo)` over `!!foo`. We just need to be careful never ever to say `new Boolean(foo)`, because boolean objects are absurd. Probably not important to make that switch in this PR, though :)